### PR TITLE
Docs/forces tutorial

### PR DESCRIPTION
**Pull request by jmlarson1**

### CODE REVIEW

**Comment by jmlarson1** [comment]
If this helps in a review: 
https://libensemble.readthedocs.io/en/develop/tutorials/executor_forces_tutorial.html
now looks like:
https://libensemble.readthedocs.io/en/docs-forces_tutorial/tutorials/executor_forces_tutorial.html

File: docs/tutorials/executor_forces_tutorial.rst [hunk 1]
```diff
@@ -182,46 +123,32 @@ Write an alternative Calling Script similar to above, but with the following dif
...
-   can give any string in place of ``__file__``.
+      .. literalinclude:: ../../libensemble/tests/functionality_tests/test_executor_forces_tutorial_2.py
+          :language: python
+          :start-at: # Starts one persistent generator
+          :end-at: )
 
-   .. code-block:: python
-       :linenos:
-       :lineno-start: 72
+   **Soln 4.** End your script in the following manny to save the output based
```
Comment (shuds13) [inline docs/tutorials/executor_forces_tutorial.rst:145]: the following manny

File: libensemble/tests/functionality_tests/test_executor_forces_tutorial_2.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+from libensemble.alloc_funcs.start_only_persistent import only_persistent_gens as alloc_f
+from libensemble.executors import MPIExecutor
+from libensemble.gen_funcs.persistent_sampling import persistent_uniform as gen_f
+from libensemble.specs import AllocSpecs, ExitCriteria, GenSpecs, LibeSpecs, SimSpecs
+
+logger.set_level("DEBUG")
+
+if __name__ == "__main__":
+    # Initialize MPI Executor
+    exctr = MPIExecutor(custom_info={"mpi_runner": "openmpi"})
```
Comment (shuds13) [inline libensemble/tests/functionality_tests/test_executor_forces_tutorial_2.py:17]: Are we somehow running this only in the openmpi job?

**Comment by shuds13** [comment]
I'm inclined to remove this from sine tutorial - what do you think?
", while input/output and other fields that libEnsemble needs to operate the generator are placed outside user."

**Comment by shuds13** [comment]
In sine tutorial it referes to sine_gen.py - but we just call this gen.py. This is what its called on main at least - did we never pull our fixes into develop?

File: docs/tutorials/executor_forces_tutorial.rst (hunk 1 above)
Comment (jmlarson1) [reply docs/tutorials/executor_forces_tutorial.rst]: Typo fixed.

File: libensemble/tests/functionality_tests/test_executor_forces_tutorial_2.py (hunk 2 above)
Comment (jmlarson1) [reply libensemble/tests/functionality_tests/test_executor_forces_tutorial_2.py:17]: This came from copying/pasting the content from `docs/tutorials/executor_forces_tutorial.rst` to this test file. (see line 200 of the `develop` version of `docs/tutorials/executor_forces_tutorial.rst`)

This test is being invoked on Github actions with local comms, 4 workers. But (in general) openmpi is not available in the GH testing environment by default.

**Comment by jmlarson1** [comment]
> I'm inclined to remove this from sine tutorial - what do you think? ", while input/output and other fields that libEnsemble needs to operate the generator are placed outside user."

Addressed in bceaf55e0

**Pull request merged**